spec/helpers/application_helper_spec.rb

```diff
@@ -76,6 +76,7 @@
     before do
       allow(ApplicationConfig).to receive(:[]).with("APP_PROTOCOL").and_return("https://")
       allow(ApplicationConfig).to receive(:[]).with("APP_DOMAIN").and_return("dev.to")
+      SiteConfig.app_domain = "dev.to"
```

Comment:
Do we need to unset this so that it doesn't leak into other tests?

Reply:
Ideally all functionality is independently flexible such that it can take a different `SiteConfig.app_domain`, so as long as the test itself doesn't test the specific value here without first setting it, it's probably okay if it changes from test to test.

... But that might not be ideal in terms of our overall test suite consistency. I'm not sure if it's better to unset values like this or to be resilient to changing values 🤷

Reply:
I'd probably replace it with a mock:

allow
```suggestion
      alllow(SiteConfig).to receive(:app_domain).and_return("dev.to")
```

This is not only about our codebase being resilient to changes, but also our test environment, i.e. will this work with different Travis workers running different sets of tests due to Knapsack and changing this value non-deterministically?